types/check.ts

```diff
@@ -764,3 +766,86 @@ websocket1.addEventListener('message', evt => {
 	assert<MessageEvent>(evt);
 	assert<string>(evt.data);
 });
+
+/**
+ * WORKTOP/MODULES
+ */
+
+const worker1: ModuleWorker = {
+	async fetch(req, env, ctx) {
+		assert<Request>(req);
+		assert<IncomingCloudflareProperties>(req.cf);
+
+		assert<Bindings>(env);
+
+		// @ts-expect-error
+		let foobar = env.foobar;
+
+		assert<Function>(ctx.waitUntil);
+		assert<Function>(ctx.passThroughOnException);
+
+		return new Response;
+	},
+	async scheduled(event, env, ctx) {
+		// @ts-expect-error
+		assert<CronEvent>(event);
+		assert<string>(event.cron);
+		assert<number>(event.scheduledTime);
+		// @ts-expect-error
+		assert<undefined>(event.waitUntil);
+		assert<Function>(ctx.waitUntil);
+
+		assert<Bindings>(env);
+	}
+};
+
+interface CustomBindings extends Bindings {
+	DATAB: KV.Namespace;
+	SECRET: string;
+	// @ts-expect-error
+	COUNT: number;
+}
+
+const worker2: ModuleWorker<CustomBindings> = {
+	fetch(req, env, ctx) {
+		// @ts-expect-error
+		assert<Bindings>(env);
+		// @ts-expect-error
+		assert<CustomBindings>(env);
+
+		assert<string>(env.SECRET);
+		assert<KV.Namespace>(env.DATAB);
+		assert<number>(env.COUNT);
+
+		// @ts-expect-error
+		let foo = env.missing;
```

Comment:
This is an error because a `missing` property was not defined in the `CustomBindings` interface.